### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by beatrizserrano**
Minor changes, mostly fixing typos

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -860,19 +857,19 @@ sponsors:
...
+    - `new-url` can be either an internal relative URL ("/admin/config/tool-dependencies/", note the missing `src` at the front and the missing `index.md` at the end) or point outside the hub.
     - You are encouraged to also drop all text from the page.  Optionally, you can include something like:
 
       ```
       This tutorials index page has been merged with the new [Learn Galaxy](/learn/)
       hub page, please check there for up-to-date references to tutorials.
       ```
 
-- :chicken: When deleting, consolidating, or renaming pages please save the urls by adding redirects to the [rewrite file](https://github.com/galaxyproject/infrastructure-playbook/blob/master/galaxyenv/templates/nginx/galaxyproject.j2#L46) in the infrastructure playbook.
+- :chicken: When deleting, consolidating, or renaming pages, please save the urls by adding redirects to the [rewrite file](https://github.com/galaxyproject/infrastructure-playbook/blob/master/galaxyenv/templates/nginx/galaxyproject.j2#L46) in the infrastructure playbook.
```
Comment (beatrizserrano) [inline CONTRIBUTING.md:868]: This link is broken and I'm not sure where the file is now or whether this line makes sense anymore

Comment (bgruening) [reply CONTRIBUTING.md:868]: It is this one here: https://github.com/galaxyproject/infrastructure-playbook/blob/master/env/galaxy/templates/nginx/galaxyproject.j2

**Review by tnabtaf — APPROVED** [review]

**Comment by tnabtaf** [comment]
Yow!  Thanks for catching all of these.  And my double spaces between sentences are going to drive you crazy.

File: CONTRIBUTING.md (hunk 1 above)
Comment (beatrizserrano) [reply CONTRIBUTING.md:868]: danke sehr!

**Comment by beatrizserrano** [comment]
> And my double spaces between sentences are going to drive you crazy.

Oh yes they will! :rofl: I will learn to live with them ;) Thanks for the review!

**Pull request merged**